src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonTypeInfo.Cache.cs

```diff
@@ -14,7 +14,7 @@ public partial class JsonTypeInfo
         /// <summary>
         /// Cached typeof(object). It is faster to cache this than to call typeof(object) multiple times.
         /// </summary>
-        public static readonly Type ObjectType = typeof(object);
+        internal static readonly Type ObjectType = typeof(object);
```

Comment:
BTW, this field was added based on https://github.com/dotnet/runtime/pull/37976#discussion_r442342658. We should verify there are no run-time optimizations that make it unnecessary (post this PR).